src/test/app/case/types/task-list/task-list.ts

```diff
@@ -0,0 +1,59 @@
+import { expect } from 'chai';
+
+import { Alert, SubTask, TaskList, Task, TaskStatus } from '../../../../../main/app/case/types';
```

Comment:
once my PR goes in, you can simplify above as `import { Alert, SubTask, TaskList, Task, TaskStatus } from 'case/types';`